server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -619,6 +622,34 @@ export function configureWebSocketServices(
 				);
 			}
 
+			// Send signal to room from broadcast-signal endpoint
+			if (httpServer !== undefined && eventEmitter !== undefined) {
+				eventEmitter.on("broadcast-signal", (_, containerUrl) => {
+					const documentId = containerUrl.split("/")[containerUrl.split("/").length - 1];
+					const tenantId = containerUrl.split("/")[containerUrl.split("/").length - 2];
+					const roomFromBroadcastSignal: IRoom = { tenantId, documentId };
+
+					const signalMessageRuntimeMessage: ISignalMessage = {
+						clientId: null, // system signal
+						content: JSON.stringify({
```

Comment:
Might be a case to make this content message into a static string saved somewhere along with some information about its purpose.